File: programmers/난이도별/level01.K번째수/junilhwang.py
def solution(array, commands):
    result = []
    for v in commands:
        i, j, k = v
        temp = array[i-1:j]
        temp.sort()
        result.append(temp[k-1])
    return result


a = [[2, 5, 3], [4, 4, 1], [1, 7, 3]]
b =	[1, 5, 2, 6, 3, 7, 4]

print(solution(b,a))

File: programmers/코딩테스트_고득점kit/4_정렬.K번째수/junilhwang.js
// https://programmers.co.kr/learn/courses/30/lessons/42748?language=javascript
function solution(array, commands) {
  return commands.map(([i, j, k]) => array.slice(i-1, j).sort((a, b) => a- b)[k-1])
}

console.log(solution([1, 5, 2, 6, 3, 7, 4], [[2, 5, 3], [4, 4, 1], [1, 7, 3]]),[5, 6, 3])
//console.log(solution([1,1,1,1,3,2,2,2,2], [[]]),[])